.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+exclude = .git,__pycache__,build,dist,versioneer.py,pcdsdevices/_version.py,docs/source/conf.py
```

Comment:
Compare with the previous exclude list:
https://github.com/pcdshub/pcdsdevices/blob/ce56c58a9fcb7e047a2efbdcbd179838e9970884/setup.cfg#L8

Reply:
Thaank you